web/client/examples/viewer/components/FullScreen.jsx

```diff
@@ -0,0 +1,51 @@
+var React = require('react');
+var {ButtonGroup} = require('react-bootstrap');
+var {Button, Glyphicon} = require('react-bootstrap');
+
+var full = React.createClass({
+    propTypes: {
+        activeKey: React.PropTypes.string
+    },
+    getInitialState() {
+        return {
+            active_full_button: false,
```

Comment:
you don't need the internal state if you use true or false for the  prop "fullscreen", instead of activeKey.